### PR TITLE
I18n sync Down & Out 03/12

### PR DESCRIPTION
**Pull request by Hamms**

### CODE REVIEW

File: apps/i18n/common/cs_cz.json [hunk 1]
```diff
@@ -675,8 +680,33 @@
...
+  "loginInfo_joinStep3": "Přejděte na stránku {url} a zadejte jejich kód k sekci: {code}.",
+  "loginInfo_joinStep4": "Jakmile stisknou tlačítko \"Go\", měly by být přidáni do vaší sekce.",
+  "loginInfo_noStudents": "*Zřejmě v této sekci nemáte žádné studenty. Přidejte některé studenty do seznamu [Manage Students]({url}) k této sekci.*",
+  "loginInfo_oauthSectionCodes": "{provider} sekce nemají 6místný kód, takže váš student nebude moci použít kódu k přihlášení.",
+  "loginInfo_resetTitle": "Resetování studentských hesel",
+  "loginInfo_resetPasswordBody": "Pokud si váš student vytvořil účet Code.org pomocí své e-mailové adresy a hesla (místo přihlášení prostřednictvím služby Google, Facebook, Microsoft nebo Clever), můžete obnovit jeho heslo na kartě [Manage Students] ({url}) z vašeho oddílu kliknutím na tlačítko \"Resetovat heslo\" ve sloupci \"Heslo\".",
+  "loginInfo_resetSecretBody": "Můžete obnovit bezpečnost pro každého ze svých studentů tím, že kliknete na kartu [Manage Students] ({url}) této sekce a dále na tlačítko Zobrazit tajné ve sloupci Heslo a zvolíte tlačítko Reset secret.",
```
Comment (Hamms) [inline apps/i18n/common/cs_cz.json]: looks like a translation broke the link in this string

File: dashboard/config/locales/authored_hints.cs-CZ.yml [hunk 2]
```diff
@@ -151,6 +191,17 @@ cs:
           než začne kreslit.
         2-3_Artist_Loops_New_9_b: Tohle V je ve skutečnosti prostě polovina čtverce.
           Jaký by byl kód k nakreslení půlky čtverce?
+      2-3 Artist Nested Loops 1_authored_hint:
+        2-3_Artist_Nested_Loops_1_a: Nezapomeňte vložit všechny šedé bloky dovnitř
+          vaší smyčky.
+      2-3 Artist Nested Loops 3_authored_hint:
+        2-3_Artist_Nested_Loops_3_a: "Potřebujete pomoc s úhly?  \n** 360/12 = 30
```
Comment (Hamms) [inline dashboard/config/locales/authored_hints.cs-CZ.yml]: the added whitespace between `**` and `360` here and below will break the formatting

**Review by Hamms — COMMENTED** [review]
couple of bad translations that we should fix on Crowdin, but generally LGTM

File: dashboard/config/locales/authored_hints.cs-CZ.yml (hunk 2 above)
Comment (bethanyaconnor) [reply dashboard/config/locales/authored_hints.cs-CZ.yml]: Are these things we should fix in this PR? Or just remove them and fix them in Crowdin?

Comment (Hamms) [reply dashboard/config/locales/authored_hints.cs-CZ.yml]: The most important thing is to fix them in Crowdin so they'll be good for the next sync. Depending on the severity of the issue, I also like to fix them manually in this PR; for that, it's generally fine to just update the synced out files directly.

Comment (bethanyaconnor) [reply dashboard/config/locales/authored_hints.cs-CZ.yml]: Fixed the ones in this file and the other you pointed out, both in the code and in Crowdin.

**Review by nkiruka — APPROVED** [review]

**Pull request merged**